README.md

```diff
@@ -1 +1,36 @@
+
+
+## 🛠 Built With <a name="built-with"></a>
+- HTML
+- CSS
+
+## <img src="https://scontent.fisb6-1.fna.fbcdn.net/v/t39.30808-1/244664695_100276912439806_3650167602209431250_n.jpg?stp=dst-jpg_p200x200&_nc_cat=111&ccb=1-7&_nc_sid=7206a8&_nc_eui2=AeHOoS20TXUIBI-cKLdq7MyDYfDGj-ys0nFh8MaP7KzScV84n31f_TIamvcf7BRqujzs8jBIhO4AIjnmOxsw6aFp&_nc_ohc=S8v8SjRuzX8AX9Ic0VN&_nc_ht=scontent.fisb6-1.fna&oh=00_AfAN8Wp-6faZ5xU0_Wa00X4SwPTDWgCh1N3eq1-Uska-eg&oe=63C2EE23" border-radisu="50%" width="30px" style="border-radius: 50%;"> Authors <a name="authors"></a>
+👤 **Syed Atril**
+
+- GitHub: [@githubhandle](https://github.com/Atril33)
+- Twitter: [@twitterhandle](https://twitter.com/AtrilSyed)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/syed-atril-831696248/)
+- Facebook [Facebook](https://web.facebook.com/profile.php?id=100073724910623)
+
+## Preview
+<div align="center">
+ <img src="https://lh3.googleusercontent.com/fife/AAbDypCuC9emFacRVh-X-Rtipas4pwJzQrx10Ti3RejRLtMqme0W4OthclViiUk4zoycgnTkmAFIJbMcM5UH0Fga2J2_5uso7jTzT8C8oiAhB1vWQfmtLknXQSDaUGe48-bADdZNqfjseUw5mCxEaz7t6Ed0_i-YrjJFpDcVuyVEg5HI5GQc0V5BsDJrKceU3oGQlYJG2AGpALqAPa7reyLEWxAIGPXol7uCLXkAzjkeEZyoAe1uUgpZ_lUNdR8FjKmBzXecffCBMlGt4o4EGgsUnRMkX87-9XPNT0_WyBiUG6J9ZQyTpJGYsVfZJyoNj2ivPnjCQdIt3OJlgb3nqExQO5MV8h0Y6T07z4KhKiPp5_aMGE_Oq8QSsy53dGkTUBKvpxeCEb8n5OFIX3RYryzsMFtG_uCOfy6_IyucJ0ybnQDyWkxlzXFB3YgiYw_HUoLZ2BNpyNC95v0NquZvTWmv-7qH-b9r8fsFhOrv9XB-GPDb82ACkKv85goErSNGB83NsdWy_ROhNm2aG-tQWUDiotfGlP40EFCY4zFHDN1piDpozh37rdZrv-YU48sYS3wN4wZLdeTltPaZjl8f4CRMk6eq5XkjCgZMtMuKiYGXmeSPcgIUM99oYVN3KQTd7aZ-pwbXYqcMgMO5Fp-qWB4jeZoVfRU4hrkOQ-UEaYp8gpqLeQTJrzI9hOPb1BFxfpMqUC3esvSXSOnOUmw_mb09l6fxYplbxNuz1dt0lquqIc1B9aCpIYZLjYLXdwmj84UIqWlizW9nB0zCjqSJqBqzOlQCr9TAHSKICGakQAg6HPVRHFQEj1hi5kIn8lP51Q3b_UOBpLVpeIosW0Zu1aCxWxieAnnD8qUc14w9AsBvrJKqq2I44h-OsM6VCRahs_p1eaoC7HqwXpmuNu8ycyMBe7MdaSc8ecqrhPeRjIzqO6Z7D1k2o0l-kQrehK6ViUwOVT1kPh86GjJgITEMKUTOrBuqNLOL3erJxkJJZ1Ztm0NytL7La78LgDFakmAsZzwG-WMj8S_Tc2xQNnbluaV19W6W1uq8g7eR0pHcPX1E-Dn4sDaObcLgttlsuJamGmPJreYXHAXH4S7roFl05_txgdIC_oYGRGTT4kRWDvxB3j3-3bsTPiyCdwXps4m3WjnGCOrhiInLU890K7Ug4G6kfi8vhDrsdJCWd5IrzygmZFS0piyenN0xQulyogI3uo00q-YGD1G319aNbo1X8pDVfLnrY8-qXEyFMRCRL81LoiG5wCQxQOiAgIAogV8aeWj2EzHVE1iMxR28ruUe76Me8w0vY2NNJj1UBH9Ne4FOBJ_k9ptV4iYSK-7ofaOL7frQ0FsXlxN_rLYJwcUlB7S_BRHhb-XrmzWwAbb0BhQ2bPkgtU49MhRKztuKqYD7zj0xyN7_rZMEi0LgeBD4QoDa7ziYU0vkjGkiBVUNuZ30VA3c9ojSYyoRk2jqlSsT17gc0NPHXoUqGF8QOQQ9kHI=w1366-h617" alt="My project Preview">
+  </div>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
Hey 👋 I love what you did on the README, especially with the profile picture 😎 but in order to make it look more professional do not remove the sections described in the Microverse Template, if you feel like a section is not relevant to the project, leave it in the README and add: `N/A` as an answer to it but your README should include all the sections. Only the FAQ ❓ is optional. 😉